packages/shared/src/components/auth/OnboardingRegistrationForm.tsx

```diff
@@ -179,6 +182,7 @@ const OnboardingRegistrationForm = ({
           label="Email"
           type="email"
           name="email"
+          focused={onboardingPapercut}
```

Comment:
Just check this doesn't break on mobile, I vaguely recall we removed it for some reason.